include/ck/utility/type.hpp

```diff
@@ -29,6 +29,9 @@ using remove_cv_t = typename std::remove_cv<T>::type;
 template <typename T>
 using remove_cvref_t = remove_cv_t<std::remove_reference_t<T>>;
 
+template <typename T>
+using remove_ptrcvref_t = typename std::remove_pointer<remove_cvref_t<T>>::type;
```

Comment:
please simply use 
```
template <typename T>
using remove_pointer_t = typename std::remove_pointer<T>::type;
```

Reply:
Fix in 420fe8925fbdf357f292f52378c6624bb3015e8d